editions/tw5.com/tiddlers/system/doc-styles.tid

```diff
@@ -285,6 +285,14 @@ ol.doc-github-contributors li {
 	overflow: hidden;
 	text-overflow: ellipsis;
 }
+.doc-op-comparison {
+	table-layout: fixed;
+	width: 80%;
+}
+.doc-op-comparison th .doc-operator {
+	background-color: unset;
+	color: #666;
```

Comment:
colors need to be used from palette values. Otherwise palette switching may make them invisible or hard to read.

Reply:
There is no sensible and semantic palette entry for this application, and I would not want to define a new one for that purpose. The color `very-muted-foreground` is too light on a gray table header background.
I could use the `foreground` palette entry with some transparency, like so
```
color: <<colour foreground>>A0;
```
but I cannot be 100% sure that this will work for all palettes.